pkg/controllers/managementuser/resourcequota/resource_quota_common.go

```diff
@@ -203,7 +203,7 @@ func getProjectContainerDefaultLimit(ns *corev1.Namespace, projectLister v3.Proj
 		return nil, nil
 	}
 	project, err := projectLister.Get(projectNamespace, projectName)
-	if err != nil || project.Spec.ResourceQuota == nil {
```

Comment:
I'm guessing here, but I think this was here because of copypaste from https://github.com/rancher/rancher/pull/36334/files#diff-19dc1cfc80346c6a1f3908d3850916091cedaa8c6ee9f66fbdcbdcfc6594d739L164 and https://github.com/rancher/rancher/pull/36334/files#diff-19dc1cfc80346c6a1f3908d3850916091cedaa8c6ee9f66fbdcbdcfc6594d739L185 . Unlike https://github.com/rancher/rancher/pull/36334/files#diff-19dc1cfc80346c6a1f3908d3850916091cedaa8c6ee9f66fbdcbdcfc6594d739L172 we're not dereferencing ContainerDefaultResourceLimit so I don't think we need the same nil check here.